src/main/java/com/example/notes/entity/NoteEntity.java

```diff
@@ -0,0 +1,20 @@
+package com.example.notes.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "notes")
+@Data
+public class NoteEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
I suggest that we create our own IDs with the UUID.randomUuid().toString() - this will give us more control over how entities are created

Reply:
Ok